miner/worker.go

```diff
@@ -251,7 +292,32 @@ func (w *worker) newWorkLoop() {
 		}
 		interrupt = new(int32)
 		w.newWorkCh <- &newWorkReq{interrupt: interrupt, noempty: noempty}
-		timer.Reset(blockRecommitInterval)
+		timer.Reset(recommitInterval)
+	}
+	// recalcuInterval recalculates the resubmitting interval upon feedback.
+	recalcuInterval := func(target float64, inc bool) {
+		var (
+			old = float64(recommitInterval.Nanoseconds())
+			new float64
```

Comment:
Please avoid using `new`. It's a Go keyword and makes the code a lot harder to follow. Perhaps instead of `old` and `new`, let's use `prev` and `next`.